src/emerge.h

```diff
@@ -196,6 +200,7 @@ class EmergeManager {
 
 	// Managers of various map generation-related components
 	// Note that each Mapgen gets a copy(!) of these to work with
+	BiomeGen *biomegen;
```

Comment:
`m_biomegen`

Reply:
I'd prefer keeping it like this for consistency with the variables below it.